spec/requests/draft_controller_spec.rb

```diff
@@ -20,4 +20,21 @@
     expect(response.status).to eq(200)
     expect(Draft.get(user, 'xxx', 0)).to eq(nil)
   end
+
+end
+
+describe DraftsController do
```

Comment:
This has been declared above in the file. New assertions should go under a new `describe` block

```
describe "#drafts" do
  it "..." do
    assertions
  end

  it "..." do
    assertions
  end
end
```